cicecore/cicedynB/dynamics/ice_dyn_evp.F90

```diff
@@ -243,6 +288,29 @@ subroutine evp (dt)
          call grid_average_X2Y('T2UF',strairy)
       endif      
 
+      if (grid_system == 'CD') then
+
+      if (.not. calc_strair) then       
+         strairxN(:,:,:) = strax(:,:,:)
+         strairyN(:,:,:) = stray(:,:,:)
+         strairxE(:,:,:) = strax(:,:,:)
```

Comment:
I would prefer that we indent everything consistently. It makes code folding much more useful in editors, and makes reading the code easier (in my opinion)..